src/whisper_at_train/run_combined_training.sh

```
@@ -82,4 +130,12 @@ python -W ignore ./run.py \
   --num-workers 8 \
   --pretrained_model ${pretrained_model} \
   --weight_decay ${weight_decay}
-  #--freeze_original_classes
\ No newline at end of file
+  #--freeze_original_classes
+
+# Check exit status
+if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)